### PR TITLE
Get player inventory

### PR DESCRIPTION
**Pull request by FernandoArevaloE**
PR contains new api for getting inventory data for given player and experience

### CODE REVIEW

File: app/src/routes/inventory.js [hunk 1]
```diff
@@ -16,4 +16,10 @@ router.get('/jackpot', async (req, res) => {
         .catch((error) => handleError(error, res));
 });
 
+router.get('/player/:id', async (req, res) => {
```
Comment (andymotagl) [inline app/src/routes/inventory.js:19]: I consider the resource should not consider player on the path. What do you think if both, player and experience are query params?

Comment (FernandoArevaloE) [reply app/src/routes/inventory.js:19]: I agree

**Comment by andymotagl** [comment]
Can the PR title be more straightforward?
I guess the title can be the description

File: app/src/routes/inventory.js (hunk 1 above)
Comment (FernandoArevaloE) [reply app/src/routes/inventory.js]: done

**Review by andymotagl — APPROVED** [review]

**Pull request merged**